package.json

```diff
@@ -20,7 +20,7 @@
     "build": "npm run clean && npm run build_utils && tsc && webpack",
     "build_utils": "tsc --build tsconfig-server.json",
     "clean": "rm -rf lib dist",
-    "docs": "typedoc --mode file --out docs",
+    "docs": "typedoc --githubPages false --out docs src/index.ts",
```

Comment:
Curious why `--githubPages=false` when we're targeting github pages?

Reply:
I'm also assuming we are using jekyll for out github pages.

Reply:
Thanks for this change!